client/src/views/Home.vue

```diff
@@ -228,12 +252,16 @@ export default {
       typing: true,
       showCursor: true,
       lightSpeed: false,
-      launchAudio: null
+      launchAudio: null,
+      backgroundMusic: null,
+      isMusicPlaying: false
     };
   },
   async mounted() {
     this.isAutoLoggingIn = true;
     this.launchAudio = new Audio(wooshAudio);
+    this.backgroundMusic = new Audio(require("../assets/audio/backgroundMusic.mp3"));
+    this.backgroundMusic.loop = true;
```

Comment:
One of the big no-nos in UX and on the internet in general is playing music for them. So I'm going to pause this PR for now.

Reply:
It's muted by default and will not play unless someone activates it. It only activates if they press the speaker button on the bottom, and they can toggle again to stop the music if they want.